companion/lib/main.ts

```diff
@@ -21,6 +21,25 @@ import logger from './Log/Controller.js'
 import { ConfigReleaseDirs } from '@companion-app/shared/Paths.js'
 import { type SyslogTransportOptions } from 'winston-syslog'
 
+const isValidIPv4 = (ip: string): boolean => {
```

Comment:
maybe this could be simplified to use https://nodejs.org/api/net.html#netisipv4input ?